### ISSUE TITLE
Tooltip component type "describe" is erasing the aria-label string assigned to the child component

### ISSUE THREAD

**Issue by williamjstanton** [Workday/canvas-kit#2405]
## 🐛 Bug Report

When I apply an `aria-label` string to a component, and then use the `describe` type tool tip to add a longer description for the component, I notice the `aria-label` string is erased. 

## To Reproduce

Please refer to my [SandBox test case](https://codesandbox.io/s/bug-canvas-tooltip-describe-variant-7k7wtr)

Scenario: I would like a text alternative set for the `sortUpIcon` in the button. I set an `aria-label` string onto the button component for that purpose. AND: I would like to use a tooltip for showing instructions about what will happen when the button is clicked.

## Expected Behavior

Accessible `name` and `description` are independent from one another. The `describe` and `muted` types of the Tooltip component should not have any impact on the `name` of a component. For setting more advanced names on a component, `aria-label` and `aria-labelledby` properties shouldn't be removed when using those types. I think a conditional check would help: 

`if type === default`
`then remove aria-label and aria-labelledby`

## Actual Results

A clear and concise description of what actually happens

My button does not have an `aria-label` anymore. I cannot set a text alternative for the `sortUpIcon` that conveys the current ascending state. 

## Browser (if applicable)

If this was a visual bug, what browser did you spot it on?

No visual impact; all browsers. 

## Link to repl or repo (highly encouraged)

Please provide a reference to a repo or branch to help us reproduce this bug.

https://codesandbox.io/s/bug-canvas-tooltip-describe-variant-7k7wtr
